include/usb_cam/usb_cam.h

```diff
@@ -36,128 +36,73 @@
 #ifndef USB_CAM_USB_CAM_H
 #define USB_CAM_USB_CAM_H
 
-#include <asm/types.h>          /* for videodev2.h */
-
-extern "C"
-{
-#include <linux/videodev2.h>
-#include <libavcodec/avcodec.h>
-#include <libswscale/swscale.h>
-#include <libavutil/mem.h>
-#include <libavutil/imgutils.h>
-}
-
-// legacy reasons
-#include <libavcodec/version.h>
-#if LIBAVCODEC_VERSION_MAJOR < 55
-#define AV_CODEC_ID_MJPEG CODEC_ID_MJPEG
-#endif
-
-#include <string>
-#include <sstream>
+#include "ros/forwards.h"
```

Comment:
```suggestion
#include <ros/forwards.h>
```

Reply:
As all these files are actually moved to `camera_driver.h`, tested and accepted.